uint8 system_error              #System Error Flag
uint8 io_error                  #IO Error Flag
uint8 swift_nap_error           #SwiftNAP Error Flag
uint8 sbp_minor_version         #SBP Minor Protocol Version Number
uint8 sbp_major_version         #SBP Major Protocol Version Number
uint8 external_antenna_present  #External Antenna Present Flag
